api/src/main/java/br/com/tinylink/api/config/SwaggerConfig.java

```diff
@@ -0,0 +1,16 @@
+package br.com.tinylink.api.config;
+
+import org.springdoc.core.models.GroupedOpenApi;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class SwaggerConfig {
+    @Bean
+    public GroupedOpenApi publicApi() {
+        return GroupedOpenApi.builder()
+                .group("api") // Nome do grupo (pode ser customizado)
+                .packagesToScan("br.com.tinylink.api") // Pacote dos seus controllers
```

Comment:
```suggestion
                .packagesToScan("br.com.tinylink.api.controllers")
```

Restrinja apenas ao grupo dos controllers, evitando assim a necessidade de varrer todo o projeto.